.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1017,6 +1017,78 @@ stages:
     - script: tracer\build.cmd CheckBuildLogsForErrors
       displayName: Check logs for errors
 
+- stage: integration_tests_azure_functions
+  condition: eq(variables['run_azure_functions_tests'], 'True')
+  dependsOn: [build_windows_tracer, generate_variables, master_commit_id]
+  variables:
+    masterCommitId: $[ stageDependencies.master_commit_id.fetch.outputs['set_sha.master']]
+
+  jobs:
+  - template: steps/update-github-status-jobs.yml
+    parameters:
+      jobs: [windows]
+
+  - job: windows
+    timeoutInMinutes: 60 #default value
+    strategy:
+      matrix: $[stageDependencies.generate_variables.generate_variables_job.outputs['generate_variables_step.integration_tests_windows_azure_functions_matrix'] ]
+    pool:
+      name: azure-windows-scale-set
+
+    steps:
+    - template: steps/clone-repo.yml
+      parameters:
+        masterCommitId: $(masterCommitId)
+    - template: steps/install-dotnet-sdks.yml
+    - template: steps/restore-working-directory.yml
+ 
+    - script: |
+        $(runtimeInstall)
+        func
```

Comment:
Why does func need to run here? Is this a smoke test that it was installed correctly?